### PR TITLE
chore/doc: update external software

### PR DESCRIPTION
**Pull request by simas232**
### Main improvements in this PR:
- update BLAST+ to 2.9.0 (#251)
- update DIAMOND to 0.9.26 (#251)

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the [development guidelines](https://github.com/SysBioChalmers/RAVEN/wiki/DevGuidelines).
- [X] Selected `devel` as a target branch
- [ ] If needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/RAVEN) about this PR

### CODE REVIEW

**Comment by simas232** [comment]
I forgot to add makeblastdb, will add this soon

**Comment by simas232** [comment]
Done, now all the new binaries and license files are in place

**Comment by Hao-Chalmers** [comment]
@simas232 could you please consider adding a test function (maybe just warp up code from one tutorial) for verifying the functionality of `homology search`, so that the reviewing process could be carries out by running this test function?

**Comment by simas232** [comment]
What about `checkInstallation`? Isn't this check sufficient?

**Comment by Hao-Chalmers** [comment]
@simas232 very nice to have these test functions.

I got follow errors when running `getDiamond`:
>Error using dispEM (line 50)
>DIAMOND blastp did not run successfully, error: 132
>
>Error in testDiamond (line 94)
>        dispEM(EM,true);

**Comment by simas232** [comment]
@Hao-Chalmers, you are right. It seems that DIAMOND blastp stops in the process, when going into `Searching alignments... `. Will try to re-compile the binary or try the older versions of DIAMOND soon

File: testing/testDiamond.m [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    %Run DIAMOND multi-threaded to use all logical cores assigned to MATLAB.
+    cores = evalc('feature(''numcores'')');
+    cores = strsplit(cores, 'MATLAB was assigned: ');
+    cores = regexp(cores{2},'^\d*','match');
+    cores = cores{1};
+    
+    %Create a temporary folder and copy both multi-FASTA files there
+    [~, ~]=system(['mkdir "' tmpDB '"']);
+    copyfile(fullfile(ravenPath,'tutorial','sce.fa'),tmpDB);
+    copyfile(fullfile(ravenPath,'tutorial','Sco_all_protein.faa'),tmpDB);
```
Comment (Hao-Chalmers) [inline testing/testDiamond.m]: Only one FASTA file is enough for the testing, the smaller one.

Comment (simas232) [reply testing/testDiamond.m]: Okay, will fix this soon

**Comment by Hao-Chalmers** [comment]
The `testDiamond` function now works.

**Pull request merged**